app/views/home/dashboard.html.erb

```diff
@@ -1,71 +0,0 @@
-<% content_for :page_title do %>
-  What's available in Publish
```

Comment:
We shouldn't just remove this feature. It exists for a reason, we should still be able to provide an overview of what's available in Publish when authenticating through signon.

Reply:
Sorry to jump in  but hope it's useful. I'm not sure I did know this would be removing the "What's in Publish" dashboard (sorry if I gave that impression at some point). While that ashboard obvs needs some discussion/work, I think we should keep it for now. I can't see how it's related to SignOn?